app/ide-desktop/lib/content/src/index.ts

```diff
@@ -335,6 +336,14 @@ function showLogs() {
 // @ts-ignore
 window.showLogs = showLogs
 
+function showPerformanceReport() {
+    const report = new performanceLogger.Report()
```

Comment:
Why this function has upper-case name ?

Reply:
Is that not the JS/TS default style? I just noticed we do not have a TS/JS style guide. We should change that.

Reply:
Sorry, I was 100% sure it's Rust. It's OK, sorry for the confusion!